wsd/AdminModel.cpp

```diff
@@ -1159,6 +1159,25 @@ void AdminModel::getMetrics(std::ostringstream &oss)
     oss << "error_unauthorized_request " << UnauthorizedRequestException::count << "\n";
     oss << "error_service_unavailable " << ServiceUnavailableException::count << "\n";
     oss << "error_parse_error " << ParseError::count << "\n";
+    oss << std::endl;
+
+    int tick_per_sec = sysconf(_SC_CLK_TCK);
+    // dump document data
+    for (const auto& it : _documents)
+    {
+        const Document &doc = *it.second;
+        std::string suffix = "{pid=" + std::to_string(doc.getPid()) + "} ";
+        oss << "doc_host" << suffix << doc.getHostName() << "\n";
+        oss << "doc_key" << suffix << doc.getDocKey() << "\n";
```

Comment:
OpenMetric does not allow values to be an string: https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#values
(This should be an Label).